srcpkgs/textosaurus/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'textosaurus'
+
+pkgname=textosaurus
+version=0.9.7
+revision=1
+build_style=qmake
+configure_args=build.pro
+hostmakedepends="git pkg-config qt5-svg-devel qt5-qmake"
+makedepends="qt5-tools qt5-svg-devel"
+depends="qt5-svg"
+short_desc="Simple cross-platform text editor based on Qt and Scintilla"
+maintainer="Leandro Vital <leavitals@gmail.com>"
+license="GPL-3.0"
+homepage="https://github.com/martinrotter/textosaurus"
+
+do_fetch() {
+	git clone --branch ${version} ${homepage}.git $wrksrc
+	cd ${wrksrc}
+	git submodule sync
+	git submodule update --init --recursive
```

Comment:
Because https://github.com/martinrotter/textosaurus/issues/29

I can understand why you used `git` but we don't do that in Void.
I revised your template, I propose this one instead:
````
# Template file for 'textosaurus'
pkgname=textosaurus
version=0.9.7
revision=1
_scommit=24d3deb8bea74458224a2cdfc384a6971822448e
build_style=qmake
hostmakedepends="pkg-config qt5-tools"
makedepends="qt5-svg-devel"
depends="qt5-svg"
short_desc="Simple cross-platform text editor based on Qt and Scintilla"
maintainer="Leandro Vital <leavitals@gmail.com>"
license="GPL-3.0-only"
homepage="https://github.com/martinrotter/textosaurus"
distfiles="https://github.com/martinrotter/textosaurus/archive/${version}.tar.gz
 https://github.com/martinrotter/scintilla/archive/${_scommit}.tar.gz"
checksum="73778d2e9de4cc4cf233f3b2832d9323e6586563746c7a2352813da62253ce18
 f9d3e1716417eacbe8128df2140a303a75734e9bf53731763d9c7be264249e75"

if [ -n "$CROSS_BUILD" ]; then
	hostmakedepends+=" qt5-svg-devel"
fi

post_extract() {
	cp -Trp ../scintilla-${_scommit} src/libtextosaurus/3rd-party/scintilla
}

````

Reply:
@lvitals thank you for pointing that out!